src/irmin/import.ml

```diff
@@ -40,6 +40,8 @@ module List = struct
   include List
   (** @closed *)
 
+  (*  let mem (s : string) l = mem s l *)
```

Comment:
This is needed to detect an actual bug -- we are comparing parents' commits with `(=)` in `Commit.merge` which might be wrong.